scripts/ci/run-test-msbuild.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+OLD_CWD=`pwd`
+TESTCMD=`realpath ${TESTCMD}`
+
+${TESTCMD} --label=remove-mono-temp-dir --timeout=1m rm -rf /tmp/xplat-master
+${TESTCMD} --label=autogen --timeout=15m ./autogen.sh --prefix=/tmp/mono-from-source
+${TESTCMD} --label=make --timeout=30m make PROFILE=net_4_x -j 2
```

Comment:
you don't need to recompile mono here, you can just pass the desired prefix to autogen.sh in run-jenkins.sh.

Reply:
Is there a way to do this and still install to a custom prefix? Needing someone to use run-jenkins to run the test script is undesirable

Reply:
I still don't understand what to do here